Library/Formula/fizsh.rb

```diff
@@ -0,0 +1,24 @@
+class Fizsh < Formula
+  homepage "https://github.com/zsh-users/fizsh"
+
+  head "https://github.com/zsh-users/fizsh", :using => :git
+
+  stable do
+    url "https://downloads.sourceforge.net/project/fizsh/fizsh-1.0.8.tar.gz"
+    sha1 "515f8828c8bd9f2da1e2716bb3d60727e2f26e90"
+  end
+
+  depends_on "zsh"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    assert_equal "fizsh, version #{version}", `#{bin}/fizsh -v`.strip
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!